hack/docs/generate.sh

```diff
@@ -19,6 +19,7 @@ source "$(dirname "$0")/../../test/common.sh"
 build_kb
 
 # ensure that destroy succeed
+chmod -R +w docs/book/src/multiversion-tutorial/testdata/project/
```

Comment:
Ensure that you run
- make install -. so that you have the bin with the code from master branch installed locally
- then, make generate-docs - so that you regenerated the docs based on your changes.